ports/minidoom/port.json

```diff
@@ -0,0 +1,30 @@
+{
+    "version": 3,
+    "name": "minidoom.zip",
+    "items": [
+        "MiniDoom.sh",
+        "minidoom"
+    ],
+    "items_opt": null,
+    "attr": {
+        "title": "MiniDoom",
+        "porter": [
+            "Zehmaluco"
+        ],
+        "desc": "MiniDOOM is very short tribute game based on Doom 1. It transforms the classic FPS into a 2D action platformer.  Collect keys and battle your way through 5 stages with the classic Doom weapons, powerups and enemies.",
+        "desc_md": null,
+        "inst": "Get the game on https://gamejolt.com/games/mini-doom/206421, place Mini Doom V1.3.exe in the ports/minidoom/gamedata/ directory.",
+        "inst_md": "Get the game on the [MiniDoom page]https://gamejolt.com/games/mini-doom/206421), place Mini Doom V1.3.exe in the ports/minidoom/gamedata/ directory.",
```

Comment:
Markdown link is missing a bracket, wont work like this

Reply:
There's a self extract code done by Fraxinus88 to solve this issue. 😄

Reply:
No I meant the markdown part, [link](url). Your missing the first bracket. If you copy the md-instructions in a markdown viewer you will see what I mean :)

Reply:
Fix it. :D